frontend/src/lib/apiClient.js

```diff
@@ -102,6 +102,19 @@ const createRecipe = recipe => apiClient.post('api/v1/recipes', {
   },
 });
 
+// COMMENTS
+
+const getComments = recipeId => apiClient.get(`api/v1/recipes/${recipeId}`);
```

Comment:
You'll still want `/comments` at the end of this. And you may not need this method at all, now that I look deeper, since you're returning the comments with the recipe.